GameDataParser/Parsers/RecipeParser.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections.Generic;
+using System.Xml;
+using GameDataParser.Crypto.Common;
+using GameDataParser.Files;
+using Maple2Storage.Types.Metadata;
+
+namespace GameDataParser.Parsers
+{
+    class RecipeParser : Exporter<List<RecipeMetadata>>
+    {
+        public RecipeParser(MetadataResources resources) : base(resources, "receipe") { }
```

Comment:
I noticed table is called "mastery**receipe**" but I don't think we should on purpose use typos.